package.json

```
@@ -51,5 +60,69 @@
   "files": [
     "dist",
     "src"
-  ]
+  ],
+  "release": {
+    "branches": [
+      "main",
+      { "name": "next", "prerelease": "rc" },
+      { "name": "beta", "prerelease": true },
+      { "name": "alpha", "prerelease": true }
+    ],
+    "plugins": [
+      "@semantic-release/commit-analyzer",
+      "@semantic-release/release-notes-generator",
+      [
+        "@google/semantic-release-replace-plugin",
+        {
+          "replacements": [
+            {
+              "files": [
+                "src/imgix.js"
+              ],
+              "from": "const VERSION = \".*\"",
+              "to": "const VERSION = \"${nextRelease.version}\"",
```

Fix typo, this lib uses a `var` not a `const` declaration for the version.
```suggestion
              "from": "var VERSION = \".*\"",
              "to": "var VERSION = \"${nextRelease.version}\"",
```